### ISSUE TITLE
About the implementation of Post-Compensation Strategy

### ISSUE THREAD

**Issue by jiequancui** [hyperconnect/LADE#1]
Hi, I'm very interested in your paper.

As seen from the paper, PC Softmax seems to be a strong baseline.

Can you provide the implementation of the PC softmax (mainly about how to calculate the p_{s}(y) and p_{t}(y) )?

**Comment by wade3han** [comment]
Hi. You can get the prior distribution from the dataset you are using. 
e.g. For p_{s}(y), it belongs to the prior distribution from train set (source dataset).

**Comment by jiequancui** [comment]
Big thanks

**Comment by yaoqi-zd** [comment]
> Hi. You can get the prior distribution from the dataset you are using.
> e.g. For p_{s}(y), it belongs to the prior distribution from train set (source dataset).

Hi, does this means I must know the target label distribution in advance before making prediction on target dataset?

**Comment by wade3han** [comment]
Yes, and we should predict the distribution of the target label using label shift methods or sampling such as Monte Carlo approach. Integrating our concept with the idea of predicting the target label distribution is an engaging future work. :)

**Comment by 22ema** [comment]
Hi, Can I use PC Softmax in the inference model.. My model trained balance dataset. but real world dataset is imbalanced... when inference with my model, I get bad precision..

**Comment by juice500ml** [comment]
@22ema Please make it a separate issue with a more detailed description for it. Thanks.
